src/components/renderers.js

```diff
@@ -3,199 +3,129 @@ import PropTypes from 'prop-types';
+  /**
+   * Returns a number between 0 and 100 (percentage), where the first gradient color stops.
+   * @default DEFAULT_GRADIENT_STOP
+   */
+  getGradientStop() {
+    return this.props.item.gradientStop ? this.props.item.gradientStop : DEFAULT_GRADIENT_STOP;
+  }
 
-    if (background !== undefined || border !== undefined) {
-      this.setState({
-        style: {
-          ...this.state.style,
-          ...border,
-          background
-        }
-      });
-    }
+  /**
+   * If the colors in the gradient should be reversed.
+   * Default order of the colors in the gradient: [brighter item.color, item.color]
+   * @default DEFAULT_REVERSE_DIRECTION
+   */
+  getReverseDirection() {
+    return this.props.item.reverseDirection ? this.props.item.reverseDirection : DEFAULT_REVERSE_DIRECTION;
   }
 
   /**
-   * Create a linear gradient using the received color and a color obtained adjusting the brightness
-   * of that color using props.gradientBrightness. The order is [adjusted color, received color];
-   * this order can be reversed using reverseDirection.
-   * @param {string} color of the item
-   * @param {boolean} reverseDirection used to reverse the order of the colors in the gradient.
+   * Create a linear gradient using the base color(calls getGradientColor) and a color obtained adjusting
+   * the brightness of that color using getGradientBrightness(). The default order of the colors is
+   * [brighter color, color]; this order can be reversed if getReverseDirection() is true.
+   * 
+   * By default, the background of an item uses a gradient, this method should be overriden if this behaviour is not wanted.
    * @returns {string} linear gradient
    */
-  getGradient(color, reverseDirection) {
+  getBackgroundGradient() {
     let colors = [
-      Color(color)
-        .lightness(this.props.gradientBrightness)
+      Color(this.getGradientColor())
+        .lightness(this.getGradientBrightness())
         .hexString(),
-      color
+      this.getGradientColor()
     ];
-    if (reverseDirection) {
+    if (this.getReverseDirection()) {
       colors.reverse();
     }
-    return (
-      'linear-gradient(' +
-      colors[0] +
-      ' ' +
-      (this.props.gradientStop === undefined ? '' : this.props.gradientStop + '%') +
-      ', ' +
-      colors[1] +
-      ')'
-    );
+
+    return ('linear-gradient(' + colors[0] + ' ' + this.getGradientStop() + '%, ' + colors[1] + ')');
   }
 
   /**
-   * If this.props.useGradient true, create gradient using props.color; otherwise return the color.
-   * @returns background style
+   * Returns the style of the item.
    */
-  getBackground() {
-    if (this.props.useGradient) {
-      return this.getGradient(this.props.color, this.props.reverseDirection);
-    } else {
-      return this.props.color;
+  getStyle() {
+    // subtract 10 because of the margin (see rct9k-items-inner class in style.css)
+    const itemHeight = this.props.itemHeight ? this.props.itemHeight - 10 : 'auto';
```

Comment:
externalize in function `getItemHeight()`, that the user may override

Reply:
Done.